blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -213,6 +252,32 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        # LE devices do not appear to expose certain properties like uuids until connect to at least once.
+        # show generic connect for these as we will not show any way to connect otherwise.
+        device_uuids = self.SelectedDevice['UUIDs']
+        show_generic_connect = not device_uuids
+        for uuid in device_uuids:
+            if ServiceUUID(uuid).short_uuid in (
+                AUDIO_SOURCE_SVCLASS_ID, AUDIO_SINK_SVCLASS_ID, GENERIC_ACCESS_SVCLASS_ID,
+                HANDSFREE_AGW_SVCLASS_ID, HEADSET_SVCLASS_ID, HID_SVCLASS_ID
```

Comment:
>I'm sorry but I have to throw in yet another discovery. 😆

Now I am sure, you are doing this intentionally 😜 

>I think we're still missing HANDSFREE_SVCLASS_ID here

You are correct, missed this one. Fixed locally.

>The short UUID for struct hog_profile is 0x1812 (HOG_UUID16 in the BlueZ code).

Which translates to the name "Human Interface Device" but we do not have it listed otherwise in the Sdp.py. Will add to the list as ``0x1812``.

>The MIDI_UUID is 03B80E5A-EDE8-4B33-A751-6CE34EC4C700 and as it's no reserved UUID there is no 16-bit version. The profile was added in BlueZ 5.44. I see three possible solutions for this:

Fourth option, add a FIXME and wait for someone to report it not working 😸. It would be best if we had a full dump of the properties of one of these devices so we can see.